railties/lib/rails/generators/rails/app/templates/config/environments/development.rb.tt

```diff
@@ -12,6 +12,9 @@ Rails.application.configure do
   # Show full error reports.
   config.consider_all_requests_local = true
 
+  # Enable server timing
+  config.server_timing = true
```

Comment:
What about to make this default for development (also default for framework defaults "6.1" )? That will make it easier to use (it will kick-in on its own once "6.1" defaults are loaded) for upgraded apps as well.

https://github.com/rails/rails/blob/61306d74759faa2efac3f3e0a454187d27befeee/railties/lib/rails/application/configuration.rb#L159

Reply:
@simi I think the most sensible is to make this the default on development only.  I agree this would be a nice default for apps upgrading to 6.1+. Let's see what people in the core and committers team think about it.

Reply:
My idea is to make it default in development only, but without need of explicit configuration entry.